lib/resolvers/event_query/eventsByOrganization.js

```diff
@@ -58,5 +58,11 @@ module.exports = async (parent, args) => {
     .populate('tasks')
     .populate('admins', '-password');
 
+  eventResponse.forEach((event) => {
+    event.registrants = event.registrants.filter(
+      (registrant) => registrant.status === 'ACTIVE'
```

Comment:
hey @guptapriyanshu7, can you put `ACTIVE` in the constants file and call it from there.

Reply:
done,